c172-checklists.xml

```diff
@@ -2769,9 +2802,15 @@
                 <scale>2.7686</scale>
             </marker>
             <condition>
-                <property>sim/model/c172p/securing/tiedownR-visible</property>
+                <or>
+                    <property>sim/model/c172p/securing/pitot-cover-visible</property>
+                    <not><property>/sim/model/c172p/securing/allow-securing-aircraft</property></not>
```

Comment:
This should be the opposite, right? We want this item to be considered only if securing is allowed AND if the pitot tube is NOT visible yet.